bin/downgrade

```diff
@@ -111,34 +111,14 @@ read_selection() {
 }
 
 prompt_to_ignore() {
-  local pkg ans ln
+  local pkg ans
 
   for pkg; do
-    grep -Eq '^IgnorePkg.*( |=)'"$pkg"'( |$)' "$PACMAN_CONF" && return 0
-
+    pacignore check --pacman-conf "$PACMAN_CONF" "$pkg" && return 0
```

Comment:
I see a potential failing case here because of `return 0`. For example, if we execute `downgrade foo bar` and `foo` is already being ignored; then the prompt for ignoring `bar` will never be executed since `prompt_to_ignore` will return 0 upon checking `foo`.

The change below should fix this and I should also add a test case for this.

```suggestion
    pacignore check --pacman-conf "$PACMAN_CONF" "$pkg" && continue
```

Reply:
Thanks, changed it